client/dashboard/docs/data-library.md

```diff
@@ -40,8 +40,7 @@ export const fetchSite = async ( id: string ): Promise< Site > => {
 		return Promise.reject( new Error( 'Site ID is undefined' ) );
 	}
 	return await wpcom.req.get( {
-		path: `/sites/${ id }?http_envelope=1&fields=ID,URL,name,icon,subscribers_count,plan,active_modules,options`,
-		apiNamespace: 'rest/v1.1',
+		path: `/sites/${ id }?fields=ID,URL,name,icon,subscribers_count,plan,active_modules,options`,
```

Comment:
One more thing: I've seen `fields` being passed as a property of an object in the second argument of `wpcom.req.get()` calls. One more thing to make consistent? Hopefully, if we're fully consistent, we can actually simplify some of the APIs and the underlying implementations at some point.

Reply:
Yes, that's another thing to make consistent. I see many examples of code like this:
```js
wp.req.get( {
  path: '/imports/analyze-url?site_url=' + encodeURIComponent( domain ),
  apiNamespace: 'wpcom/v2'
} )
```
which could be:
```js
wp.req.get(
  { path: '/imports/analyze-url', apiNamespace: 'wpcom/v2' },
  { site_url: domain }
)
```